github-repo-manager/src/main/java/com/axonivy/github/GitHubMissingCodeOfConductDetector.java

```diff
@@ -0,0 +1,34 @@
+package com.axonivy.github;
+
+import java.io.IOException;
+
+public class GitHubMissingCodeOfConductDetector extends GitHubFileDetector {
+
+	public static void main(String[] args) throws IOException {
+		new GitHubMissingCodeOfConductDetector().checkMissingFile();
+	}
+
+	public GitHubMissingCodeOfConductDetector() throws IOException {
+		super();
+	}
+
+	@Override
+	protected String getRequiredFilePath() {
```

Comment:
I would like to see only one main class and one build for all three files LICENCE, SECURITY, CODE_OF_CONDUCT.
Instead of using inheritance for each file, I would prefer that you provide a list of constant objects to the main class. In the constant object you can provide the meta information for each file. You could use a record like
public record(String requiredFilePath, String pullRequestTitle, String branchName, String commitMessage) {}